certified-connectors/Muhimbi PDF Converter Online/apiProperties.json

```diff
@@ -1,36 +1,34 @@
-{
-  "properties": {
-    "connectionParameters": {
-      "token": {
-        "type": "oauthSetting",
-        "oAuthSettings": {
-          "identityProvider": "oauth2",
-          "clientId": "001E677C-8056-47B1-B415-A461F2230223",
-          "scopes": [
-            "Flow"
-          ],
-          "redirectMode": "Global",
-          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
-          "properties": {
-            "IsFirstParty": "False"
-          },
-          "customParameters": {
-            "authorizationUrl": {
-              "value": "https://api.muhimbi.com/auth/pages/authorize.aspx"
-            },
-            "tokenUrl": {
-              "value": "https://api.muhimbi.com/auth/pages/gettoken.aspx?grant_type=authorization_code"
-            },
-            "refreshUrl": {
-              "value": "https://api.muhimbi.com/auth/pages/gettoken.aspx?grant_type=refresh_token"
-            }
-          }
-        }
-      }
-    },
-    "iconBrandColor": "#2B4D5A",
-    "capabilities": [],
-    "publisher": "Muhimbi Ltd",
-	"stackOwner": "Muhimbi Ltd"
-  }
-}
\ No newline at end of file
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "001E677C-8056-47B1-B415-A461F2230223",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://api.muhimbi.com/auth/pages/authorize.aspx"
+            },
+            "tokenUrl": {
+              "value": "https://api.muhimbi.com/auth/pages/gettoken.aspx?grant_type=authorization_code"
+            },
+            "refreshUrl": {
+              "value": "https://api.muhimbi.com/auth/pages/gettoken.aspx?grant_type=refresh_token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#2b4e5b",
+    "capabilities": [],
+    "publisher": "Manoranjan Mohapatra"

```

Comment:
@mmohapatra-muhimbi please add the "stackOwner": "Muhimbi Ltd"

Reply:
Added stackowner to the file